src/content/marinaBroker.ts

```diff
@@ -180,7 +181,10 @@ export default class MarinaBroker extends Broker {
 
           if (!accepted) throw new Error('the user rejected the create tx request');
           if (!signedTxHex) throw new Error('something went wrong with the tx crafting');
-          return successMsg(signedTxHex);
+
+          const txid = await broadcastTx(selectEsploraURL(state), signedTxHex);
+          if (!txid) throw new Error('something went wrong with the tx broadcasting');
```

Comment:
would be wroth to have here a console.debug printing the signedTxHex maybe?

Reply:
also would be useful to catch the Esplora error coming out of it

Reply:
Done.

Reply:
missing console.debug of the signedTxHex

Reply:
Is there: https://github.com/bordalix/marina/blob/broadcast_tx_283/src/content/marinaBroker.ts#L185